x-pack/plugins/security_solution/public/common/containers/use_full_screen/index.tsx

```diff
@@ -13,21 +13,6 @@ import { useShallowEqualSelector } from '../../hooks/use_selector';
 import { inputsSelectors } from '../../store';
 import { inputsActions } from '../../store/actions';
 
-export const resetScroll = () => {
```

Comment:
This to fix the timeline full screen bug?

Reply:
no, it just stops the app from moving the screen, as it's not needed anymore. it's pretty jarring on smaller screens